app/scripts/modules/netflix/fastProperties/scope/fastPropertyScopeSearch.component.ts

```diff
@@ -0,0 +1,223 @@
+import './fastPropetyScopeSearch.less';
+
+import { CATEGORY_BUTTON_LIST_COMPONENT } from './categoryButtonList.component.ts';
+
+import * as _ from 'lodash';
```

Comment:
I think you're only using `_.debounce` so we can just do `import {debounce} from 'lodash';`